ipinfo/cmd_default.go

```diff
@@ -34,6 +34,7 @@ Commands:
   cache       manage the cache.
   config      manage the config.
   quota       print the request quota of your account.
+  download    download free ipinfo database files.
```

Comment:
We should put it a bit higher... like right below `splitcidr` currently.

(And also changed in the other places in the code).